### PR TITLE
fuzzers: Makefile: add explicit all target

### PR DESCRIPTION
**Pull request by kgugala**

### CODE REVIEW

File: fuzzers/Makefile [hunk 1]
```diff
@@ -8,6 +8,8 @@ FUZZONLY=N
 # Allow building without it
 BITONLY=N
 
+all:
```
Comment (litghost) [inline fuzzers/Makefile]: Also need a .PHONY for all?

Comment (kgugala) [reply fuzzers/Makefile]: I don't think so. In my tests this works fine without PHONY

Comment (litghost) [reply fuzzers/Makefile]: Please add it, see https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

The reason it is working is there is not a file named all.

Try this:
```
make clean
touch all
make all
```
It should probably do nothing

Comment (litghost) [reply fuzzers/Makefile]: Though I guess all has dependencies, so it might work out.

Comment (kgugala) [reply fuzzers/Makefile]: actually all the fuzzer's run.ok files are `all` target dependencies.

Comment (mithro) [reply fuzzers/Makefile]: You probably want to make the `all` target the default? 
```makefile
.DEFAULT_GOAL := all
```

**Pull request merged**